Test-NetStack.psm1

```diff
@@ -908,8 +910,19 @@ Function Test-NetStack {
                         Write-Host ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisSource.InterfaceIndex) ($($thisSource.IPAddress))"
                         ":: $([System.DateTime]::Now) :: [Started] N -> Interface $($thisTarget.InterfaceIndex) ($($thisSource.IPAddress))" | Out-File $LogFile -Append -Encoding utf8 -Width 2000
 
+                        $StartTime = Get-Date
                         $thisSourceResult = Invoke-NDKPerfNto1 -Server $thisSource -ClientNetwork $ClientNodes -ExpectedTPUT $Definitions.NDKPerf.TPUT
-                       
+                        $EndTime = Get-Date
+
+                        $events = (Get-EventLog System -InstanceId 0x466,0x467,0x469,0x46a -ComputerName $thisSource.NodeName)
+
+                        if($events) { 
+                            Write-Host "Caught errors on node $($thisSource.NodeName)."
+                            "Caught errors on node $($thisSource.NodeName)." | Out-File $LogFile -Append -Encoding utf8 -Width 2000
+                            $events | Select-Object Time, EntryType, InstanceID, Message | Format-Table -AutoSize | Out-File $LogFile -Append -Encoding utf8 -Width 2000
```

Comment:
Is getting the event log from a remote machine that is unstable the right way of doing this? Would we see indications of this via the cluster log?

Reply:
I have tried looking at the log generated by Get-ClusterLog, but haven't been able to find the errors that I would be looking for. These events are emitted from Microsoft-Windows-Failover-Clustering, which I don't believe would be retrievable from anywhere but the machine itself.

Reply:
I'm not sure about that specific event channel, but I thought the cluster log should provide a cluster view. We should follow up because otherwise, this check seems it might be highly inconsistent.  Perhaps check with Travis and @smoon193 what they would recommend.

Reply:
Spoke with travis about this as well, the event channel is the correct one, but will change the command so that we are not getting the event logs from the specific node being tested, as every node should have the node membership lost event for any node in the cluster.